Logger/Logger.php

```diff
@@ -84,6 +86,6 @@ public function addRecord($level, $message, array $context = []): bool
             print_r($message."\n");
         }
 
-        return parent::addRecord($level, $message, $context);
+        return parent::addRecord($level, $message, $context, $datetime);
```

Comment:
Is parent class has 4th param
![image](https://user-images.githubusercontent.com/77260133/173021014-5883863e-e228-49b7-a335-d759e2ce0801.png)

Reply:
In M2.4.4 yes it has 4 parameters.
![image](https://user-images.githubusercontent.com/38680489/173546179-fb87211c-01c2-4113-900b-7ef3d3fb12f4.png)